prologin/contest/forms.py

```diff
@@ -148,3 +148,10 @@ class CombinedContestantUserForm(MultiModelForm):
         ('user', ContestantUserForm),
         ('contestant', ContestantForm),
     ])
+
+class ContestantSemifinalEventForm(forms.Form):
+
+    def __init__(self, *args, **kwargs):
+        possible_events = kwargs.pop('possible_events')
+        super(ContestantSemifinalEventForm, self).__init__(*args, **kwargs)
```

Comment:
Use the moderl Python 3 syntax.

```suggestion
        super().__init__(*args, **kwargs)
```